scripts/icon-list.mjs

```diff
@@ -0,0 +1,67 @@
+export const enabledIcons = [
+  'flag',
```

Comment:
can we alphabetize this list?

Reply:
thank you co-pilot

Reply:
I spent like 10mins trying to figure out why I had one less line when I used it to alphabetize it, turns out we had two `branch`'s